import org.apache.hadoop.fs.PathIOException;
public class FunctionUse10399 {
public void funcUse() {
PathIOException pathioexception = new PathIOException();
pathioexception.setOperation(null);
}
}